cbits/itoa.c

```diff
@@ -4,13 +4,15 @@
 // inspired by: http://www.jb.man.ac.uk/~slowe/cpp/itoa.html //
 ///////////////////////////////////////////////////////////////
 
-#include <stdio.h>
+#include <stdint.h>
```

Comment:
IIRC, sufficiently old, but still extant versions of Windows (really Visual Studio) don't have stdint.h. This may warrant confirmation wrt. to the supported Windows releases and toolchains for Haskell.